app/javascript/packs/play_book.jsx

```diff
@@ -1,40 +1,44 @@
 // Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
 // like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
 // of the page.
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
-import Fullscreen from "react-full-screen";
-import TopToolBar from "../components/TopToolBar";
-import BottomToolBar from "../components/BottomToolBar";
-import DrawerField from "../components/DrawerField";
-import Movement from "../components/ball/Movement";
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import Fullscreen from 'react-full-screen';
+import TopToolBar from '../components/TopToolBar';
+import BottomToolBar from '../components/BottomToolBar';
+import DrawerField from '../components/DrawerField';
 
 export default class PlayBook extends Component {
   constructor(props) {
-    super();
+    super(props);
 
     this.state = {
-      isFull: false
+      isFull: false,
+      isDrawing: true,
     };
+
+    this.handleFullScreen = this.handleFullScreen.bind(this);
+    this.handleStopDrawing = this.handleStopDrawing.bind(this);
   }
 
-  goFull = () => {
+  handleFullScreen() {
     this.setState({ isFull: true });
-  };
+  }
+
+  handleStopDrawing() {
+    this.setState({
```

Comment:
if new state is based on previous one you should call setState with function
```suggestion
    this.setState(previousState => ({ isDrawing: !previousState.isDrawing }));
```